common/Field_Dimensions.hpp

```diff
@@ -40,7 +63,9 @@ struct Field_Dimensions {
     static Field_Dimensions Current_Dimensions;
 
     Field_Dimensions()
-        : Field_Dimensions(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0) {}
+        : Field_Dimensions(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0) {
+        updateGeometry();
```

Comment:
Because the no-argument constructor here calls the constructor that takes in a bunch of arguments, the `updateGeometry()` method actually ends up getting called twice.  Once because we call the multi-parameter contructor and once right afterwards.

Since there's no need to call `updateGeometry()` after initialization, let's get rid of the method and just put its contents inside of the multi-parameter constructor below.